frontend/src/components/App.tsx

```diff
@@ -75,38 +75,23 @@ const VersionInfo = () => {
     </>;
 };
 
-const SideBar = observer(() =>
-    <Layout className="sideBar" >
-        {/* Logo */}
-        <div>
-            <Link to="/">
-                {/* Logo Image */}
-                <AnimatePresence initial={false} presenceAffectsLayout >
-                    {uiSettings.sideBarOpen
-                        ? <motion.img alt="logo" key="logoExpanded" src={RedpandaLogo} {...animProps_logo} />
-                        : <motion.img alt="logo" key="logoCollapsed" src={RedpandaIcon}   {...animProps_logo} />
-                    }
-                </AnimatePresence>
-            </Link>
-        </div>
+const SideBar = observer(() => {
+    const ignoredRoutes = [''];
 
-        {/* Menu */}
-        <Content className="scroll-on-hover-y">
-            <RouteMenu />
-        </Content>
+    const sidebarItems = APP_ROUTES.filter((x) => x.icon != null)
+      .filter((x) => !ignoredRoutes.includes(x.path))
+      .map((item) => ({
+          name: item.title,
+          to: item.path,
+          icon: item.icon,
+        })
+      );
 
-        {/* Profile */}
+    return (
+    <Sidebar items={sidebarItems}>
         <UserProfile />
-
-
-        {/* Toggle */}
-        <Footer className="sideBarToggle" onClick={() => { uiSettings.sideBarOpen = !uiSettings.sideBarOpen; }}>
-            {uiSettings.sideBarOpen
-                ? <MenuFoldOutlined className="icon" />
-                : <MenuUnfoldOutlined className="icon" />}
-        </Footer>
```

Comment:
Removed the toggle for now as it was messing with styles.  I will add this back in `ui` library, so that we have the control built into nav bars.